src/SourceBuild/content/Directory.Build.targets

```diff
@@ -7,7 +7,7 @@
       <SdkFilenamePrefix>dotnet-sdk-</SdkFilenamePrefix>
     </PropertyGroup>
     <ItemGroup>
-      <SdkTarballItem Include="$(ArtifactsAssetsDir)$(SdkFilenamePrefix)*$(ArchiveExtension)" />
+      <SdkTarballItem Include="$(ArtifactsAssetsDir)**/$(SdkFilenamePrefix)*$(ArchiveExtension)" />
```

Comment:
Can this be more explicit so that we only return the one in the inner directory? Otherwise we would return both the inner and the outer one if this target runs after the `CopySdkArchive` target.

Reply:
Fixed with https://github.com/dotnet/installer/pull/18917/commits/9ccd9fef91977119e248048e95b7178e06691a86